client/src/components/Atoms/FileUpload.tsx

```diff
@@ -141,20 +143,33 @@ const FileUpload: React.FC<IFileUploadProps> = ({
       </Row>
       <Row style={{ justifyContent: 'flex-end' }}>
         {!processing?.completedAt ? (
-          <Button
-            type="submit"
-            intent="primary"
-            icon="upload"
-            disabled={
-              uploadDisabled ||
-              numSelectedFiles === 0 ||
-              formState.isSubmitting ||
-              (processing !== null && !processing.completedAt)
-            }
-            style={{ width: buttonAndTagWidth }}
-          >
-            Upload
-          </Button>
+          <>
+            {templateFileUrl && (
+              <AnchorButton
+                href={templateFileUrl}
+                icon="download"
+                rel="noopener noreferrer"
+                style={{ marginRight: '5px' }}
+                target="_blank"
+              >
+                Download Template
+              </AnchorButton>
+            )}
```

Comment:
I'm just realizing that, after you've uploaded a file, the audit admin view no longer displays the ability to download a template without first deleting the upload. (The jurisdiction admin view always gives you the ability to download a template.) A dropdown button could work here, where the primary option is "Download" and the secondary option is "Download Template"

<table>
<img alt='aa' src='https://github.com/votingworks/arlo/assets/12616928/0800c815-d4bd-4933-82b9-ff02a38e6370' width=400 />
</table>

Reply:
Hmm, interesting. Maybe just adding a third button "Download Template" to the row of buttons would be fine? Or just leaving as is - I don't think it's a huge deal

Reply:
https://github.com/votingworks/arlo/pull/1888/commits/1ef4c281344f963821a1cba763e7e22b8810b52c

Went with the third button, and updated PR screenshots accordingly